eZ/Publish/Core/settings/repository/autowire.yml

```diff
@@ -18,8 +18,3 @@ services:
     eZ\Publish\API\Repository\URLWildcardService: '@ezpublish.api.service.url_wildcard'
     eZ\Publish\API\Repository\URLAliasService: '@ezpublish.api.service.url_alias'
     eZ\Publish\API\Repository\TrashService: '@ezpublish.api.service.trash'
-
-    eZ\Publish\API\Repository\PermissionResolver:
```

Comment:
I would actually keep `*API*` here, aliasing proper Core service introduced in `inner.yml`

Reply:
> I would actually keep `*API*` here, aliasing proper Core service introduced in `inner.yml`

Reflected in #46